lib/jsdom/browser/resource-loader.js

```diff
@@ -59,12 +76,34 @@ function wrapCookieJarForRequest(cookieJar) {
   return jarWrapper;
 }
 
-function fetch(urlObj, cookieJar, referrer, callback) {
-  if (urlObj.hostname) {
-    exports.download(urlObj, null, cookieJar, referrer, callback);
+function fetch(element, urlObj, cookieJar, referrer, pool, agentOptions, userAgent, callback) {
+  let req = null;
+  if (urlObj.protocol === "data:") {
+    process.nextTick(() => {
+      const pathname = urlObj.href.match(/^data:(.+?)(?:;(base64))?,(.*)$/);
```

Comment:
pathname is not the right name for this variable. Not sure what the best name is though.